server/schema/clams.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE clams IF NOT EXISTS{
```

Comment:
the `clams` table hasn't been made in the database yet; it should be in there for this task to be finished